### PR TITLE
Document message requests

### PR DESCRIPTION
**Pull request by Dziurwa14**

### CODE REVIEW

**Comment by wingio** [comment]
lgtm

**Review by wingio — APPROVED** [review]

**Review by aamiaa — APPROVED** [review]
LGTM
(Let's Gamble, Try Merging)

File: pages/resources/channel.mdx [hunk 1]
```diff
@@ -891,6 +894,39 @@ In a managed group DM, only the managing application can add recipients.
...
 
+<RouteHeader method="PUT" url="/channels/{channel.id}/recipients/@me">
+  Update Message Request
+</RouteHeader>
+
+Returns a [DM channel](#channel-object) on success.
+
+<Alert type="warn">
+
+All consent statuses other than `ACCEPTED` are employee-only.
```
Comment (splatterxl) [inline pages/resources/channel.mdx:905]:
```suggestion
All consent statuses other than `ACCEPTED` are only usable by [Discord staff](/resources/user#user-flags).
```

File: pages/resources/channel.mdx [hunk 2]
```diff
@@ -891,6 +894,39 @@ In a managed group DM, only the managing application can add recipients.
...
+
+</Alert>
+
+###### JSON Params
+
+| Field          | Type    | Description                           |
+| -------------- | ------- | ------------------------------------- |
+| consent_status | integer | The [consent status](#consent-status) |
```
Comment (splatterxl) [inline pages/resources/channel.mdx:913]:
```suggestion
| consent_status | integer | The new [consent status](#consent-status) |
```

**Review by splatterxl — APPROVED** [review]

**Pull request merged**